packages/experimental/Expander/windows/ReactNativeExpander/ExpanderViewManager.cpp

```diff
@@ -0,0 +1,133 @@
+        nativeProps.Insert(L"headerBorderThickness", ViewManagerPropertyType::Number);
+        nativeProps.Insert(L"contentBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"contentBorderBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronForeground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronPointerOverBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronPointerOverForeground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronPressedBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronPressedForeground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronBorderThickness", ViewManagerPropertyType::Number);
+        nativeProps.Insert(L"chevronBorderBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronBorderPointerOverBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronBorderPressedBrush", ViewManagerPropertyType::Color);
+
+        return nativeProps.GetView();
+    }
+
+    void ExpanderViewManager::UpdateProperties(xaml::FrameworkElement const& view,
+        IJSValueReader const& propertyMapReader) noexcept {
+        if (auto expanderView = view.try_as<ExpanderView>()) {
+           expanderView->UpdateProperties(propertyMapReader);
+        }
+        else {
+           OutputDebugStringW(L"Type deduction for ExpanderView failed.");
+        }
+    }
+
+    // IViewManagerWithExportedEventTypeConstants
+    ConstantProviderDelegate ExpanderViewManager::ExportedCustomBubblingEventTypeConstants() noexcept {
+        return nullptr;
+    }
+
+    ConstantProviderDelegate ExpanderViewManager::ExportedCustomDirectEventTypeConstants() noexcept {
+        return [](winrt::IJSValueWriter const& constantWriter) {
+            WriteCustomDirectEventTypeConstant(constantWriter, "onChange");
+            WriteCustomDirectEventTypeConstant(constantWriter, L"Collapsing");
+            WriteCustomDirectEventTypeConstant(constantWriter, L"Expanding");
+        };
+    }
+
+    // IViewManagerWithChildren
+    int32_t ExpanderViewManager::ReplaceChild(winrt::Windows::UI::Xaml::FrameworkElement const& parent, winrt::Windows::UI::Xaml::UIElement const& oldChild, winrt::Windows::UI::Xaml::UIElement const& newChild) noexcept {
+        return 0;
+    }
+
+    void ExpanderViewManager::AddView(winrt::Windows::UI::Xaml::FrameworkElement const& parent, winrt::Windows::UI::Xaml::UIElement const& child, int64_t index) noexcept {
+        auto contentPresenter = parent.try_as<winrt::Windows::UI::Xaml::Controls::ContentPresenter>();
+        auto expanderWrapper = contentPresenter.Content();
+        auto expander = expanderWrapper.try_as<Microsoft::UI::Xaml::Controls::Expander>();
```

Comment:
Can merge these 2 lines, since you're not doing anything else with `expanderWrapper`
Also, wen using `try_as` you should be checking for null since it may occur. However, if that happened here, then we can't handle it in any smart way, so might as well crash - use `.as<>()` which will throw an exception.